CI: test step fails - paramiko can not import utils module

## What
Importing the util module from paramiko fails in the [tests/helper/sshclient.p](https://github.com/gardenlinux/gardenlinux/blob/3fb576cffdd44b19a5a5edfe2293c00b32a27cfe/tests/helper/sshclient.py#L12):

```
from paramiko.util import u
```


The test step in the `dev` GitHub action fails (sometimes).

## Logs
Log of a failing run:

```
Run ./test --container-image test kvm-amd64
ImportError while loading conftest '/gardenlinux/tests/conftest.py'.
conftest.py:13: in <module>
    from helper.sshclient import RemoteClient
helper/sshclient.py:12: in <module>
    from paramiko.util import u
E   ImportError: cannot import name 'u' from 'paramiko.util' (/opt/venv/lib/python3.11/site-packages/paramiko/util.py)
Error: Process completed with exit code 4.
```
See a failed run here: https://github.com/gardenlinux/gardenlinux/actions/runs/6638045165/job/18034005079#step:14:1

## Notes
- No obvious change found on first sight. commit that failed is completely unrelated 
- Happened also in the past if I remember correctly, a re-run fixed it in the past

--- a/tests/helper/sshclient.py
+++ b/tests/helper/sshclient.py
@@ -9,7 +9,6 @@ from binascii import hexlify
 from paramiko import SSHClient, AutoAddPolicy, RSAKey
 from paramiko.auth_handler import AuthenticationException, SSHException
 from paramiko.ssh_exception import NoValidConnectionsError
-from paramiko.util import u
 from scp import SCPClient, SCPException
 
 logger = logging.getLogger(__name__)
@@ -49,7 +48,7 @@ class RemoteClient:
                 f.write(f"{pub.get_name()} {pub.get_base64()}")
                 if comment:
                     f.write(f" {comment}")
-        hash = u(hexlify(pub.get_fingerprint()))
+        hash = hexlify(pub.get_fingerprint()).decode(encoding="utf-8")
         fingerprint = ":".join([hash[i : 2 + i] for i in range(0, len(hash), 2)])
         logger.info(f"fingerprint: {bits} {fingerprint} {filename}.pub (RSA)")
         return fingerprint

